romanesco/plugins/swift/executor.py

```diff
@@ -0,0 +1,78 @@
+import os
+import re
+import romanesco.utils
+import tempfile
+
+
+def _transform_path(inputs, taskInputs, inputId, tmpDir):
```

Comment:
I think this is specific to docker since it's mapping paths on the local filesystem to paths within the docker container.